src/server.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import path from 'path';
+
+const app = express();
+const port = 3000;
+
+// Serve static files from the dist directory
+app.use(express.static('dist'));
+
+// // Route all requests to the index.html file
+app.get('/todos', (_req, res) => {
+    res.sendFile(path.join(process.cwd(), 'dist/index.html'));
+});
```

Comment:
let's add this to the commit message. Something like
create server and serve todos index.html